apps/src/dance/blockly/blocks.js

```diff
@@ -76,6 +79,51 @@ const customInputTypes = {
       return block.getFieldValue(arg.name);
     },
   },
+  dancerPicker: {
+    addInput(blockly, block, inputConfig, currentInputRow) {
+      const options = inputConfig.options.map(option => {
+        let name = option;
+        try {
+          name = JSON.parse(name);
+        } catch {}
```

Comment:
Worth logging here? When might this happen (i'm assuming only if there's some typo in the block def)?

Reply:
Oh good question! Blocks like these, including those in other pools, can have options defined like this:
```
"options": [
        [
          "all",
          "sprites"
        ],
        [
          "aliens",
          "\"ALIEN\""
        ],
        [
          "bears",
          "\"BEAR\""
        ]
    ]
```

These values are all strings in the config but when they're added to the generated code as field values, the quotes are escaped, leaving `sprites` as a reference to a global variable and things like `"BEAR"` as a string literal that we use to find the costume group.  We need to parse so that we can get the filename for the head image. So the `try catch` here is so that all config option values can correspond to a filename without breaking on `"sprites"`.